nautobot/core/management/commands/generate_test_data.py

```diff
@@ -101,178 +100,134 @@ def _generate_factory_data(self, seed, db_name):
-        ControllerFactory.create_batch(10)
-        ControllerManagedDeviceGroupFactory.create_batch(30)
+        _create_batch(ControllerFactory, 10)
+        _create_batch(ControllerManagedDeviceGroupFactory, 30)
         # make sure we have some tenants that have null relationships to make filter tests happy
         self.stdout.write("Creating Tenants without Circuits, Locations, IPAddresses, or Prefixes...")
-        TenantFactory.create_batch(10, using=db_name)
+        _create_batch(TenantFactory, 10)
         # TODO: nautobot.tenancy.tests.test_filters currently calls the following additional factories:
-        # UserFactory.create_batch(10)
-        # RackFactory.create_batch(10)
-        # RackReservationFactory.create_batch(10)
-        # ClusterTypeFactory.create_batch(10)
-        # ClusterGroupFactory.create_batch(10)
-        # ClusterFactory.create_batch(10)
-        # VirtualMachineFactory.create_batch(10)
+        # _create_batch(UserFactory, 10)
+        # _create_batch(RackFactory, 10)
+        # _create_batch(RackReservationFactory, 10)
+        # _create_batch(ClusterTypeFactory, 10)
+        # _create_batch(ClusterGroupFactory, 10)
+        # _create_batch(ClusterFactory, 10)
+        # _create_batch(VirtualMachineFactory, 10)
         # We need to remove them from there and enable them here instead, but that will require many test updates.
 
-        self._output_hash_for_factory_models(
-            factories=[
-                CircuitFactory,
-                CircuitTerminationFactory,
-                CircuitTypeFactory,
-                ContactFactory,
-                ControllerManagedDeviceGroupFactory,
-                ControllerFactory,
-                DeviceFactory,
-                DeviceFamilyFactory,
-                DeviceRedundancyGroupFactory,
-                DeviceTypeFactory,
-                ExternalIntegrationFactory,
-                IPAddressFactory,
-                LocationFactory,
-                LocationTypeFactory,
-                ManufacturerFactory,
-                NamespaceFactory,
-                PlatformFactory,
-                PrefixFactory,
-                ProviderFactory,
-                ProviderNetworkFactory,
-                RIRFactory,
-                RoleFactory,
-                RouteTargetFactory,
-                SoftwareImageFileFactory,
-                SoftwareVersionFactory,
-                StatusFactory,
-                TagFactory,
-                TeamFactory,
-                TenantFactory,
-                TenantGroupFactory,
-                VLANFactory,
-                VLANGroupFactory,
-                VRFFactory,
-            ]
-        )
-
-    def _output_hash_for_factory_models(self, factories):
-        """Output a hash of the IDs of all objects in the given factories' model.
-
-        Used for identifying factory determinism problems in unit tests. Only prints if GITHUB_ACTIONS environment variable is set to "true".
```

Comment:
Can we keep this docstring to make it more obvious why we're outputting these hashes? Maybe keep this as a separate method so we can have the docstring here?

Reply:
Added a `--print-hashes` option to the command with appropriate help text; keeping the current behavior of `$GITHUB_ACTIONS=true` automatically enabling hash printing.